src/btree/bt_split.c

```diff
@@ -972,6 +980,9 @@ __split_parent(WT_SESSION_IMPL *session, WT_REF *ref, WT_REF **ref_new,
 		parent_decr += sizeof(WT_REF);
 	}
 
+	/* We freed the reference that was split in the loop above. */
+	ref = NULL;
+
```

Comment:
@michaelcahill: not sure about the point of this line? It's still valid if we wanted to use it (and lint wants to know why we're doing this if we don't ever use `ref` again).